IFC4x3/Templates/Object Connectivity/Spatial Structure/Spatial Interference/Spatial Interference With Zones/Documentation.md

```diff
@@ -0,0 +1,9 @@
+
+The _Spatial Interference with Zone_ concept is an extension of the _Spatial Inteference_ concept which defines the relationship of spatial elements, such as facilities (_IfcFacility_ & specialised subtypes) or faclity parts (_IfcFacilityPart_ with domain specific predefined types) that interfere or interface with other spatial elements across discipline spatial hierarchy branches, and extends the dataset with an related inteference _IfcSpatialZone_ via the _IfcRelInterferesElements.InterferenceSpace_ attribute. The realizing _IfcSpatialZone_ should always have the _PredefinedType_ value set to INTERFERENCE.
+
+The addition of a realizing _ifcSpatialZone_ extends the functionality of this relationship with the following:
+- The ability to attach property sets to the co-engineering (or interference) zone.
+- Allows overlapping footprint or body geometry to remain relevent to the _IfcSpatialElement_s and then define an explicit shared  footprint or body geometry.
```

Comment:
```suggestion
- Allows overlapping footprint or body geometry to remain relevant to the _IfcSpatialElement_s and then define an explicit shared  footprint or body geometry.
```

Reply:
Sentence feels a little bit strange to me. Could it be "Allows explicit definition of the shared footprint or body geometry representing the zone where the _IfcSpatialElement_s interfere" or something similar?